src/envConfig.ts

```diff
@@ -14,6 +14,16 @@ function asCorePluginInit<T>(cleaner: Cleaner<T>): Cleaner<T | false> {
   }
 }
 
+const asEvmApiKeys = asObject({
+  alethioApiKey: asOptional(asString, ''),
+  amberdataApiKey: asOptional(asString, ''),
+  blockchairApiKey: asOptional(asString, ''),
+  evmScanApiKey: asOptional(asArray(asString), () => []),
+  gasStationApiKey: asOptional(asString, ''),
+  infuraProjectId: asOptional(asString, ''),
+  quiknodeApiKey: asOptional(asString, '')
```

Comment:
Where is `bscscanApiKey`, `ftmscanApiKey`, and `polygonscanApiKey`?

Reply:
Removed as they are deprecated and we have been consistently using `evmScanApiKey` which replaced all of those